mjpeg-view/build.gradle

```diff
@@ -32,12 +32,8 @@ android {
 }
 
 dependencies {
-    testImplementation 'junit:junit:4.13.2'
```

Comment:
no tests 😣

Reply:
There are no, that's why it's noise

Reply:
yeah, it was just a sad comment, nothing to do here